hera_mc/cm_sysutils.py

```diff
@@ -369,3 +340,38 @@ def system_comments(self, system_kw='System', kword='all'):
         comments = '\n'.join(rows)
 
         return comments
+
+    def _get_apriori_antenna_set(self):
+        aa_set = []
+        for val in self.session.query(cm_partconnect.AprioriAntenna.antenna).distinct():
+            aa_set.append(val[0])
+        return aa_set
+
+    def get_apriori_for(self, ant, at_date='now'):
+        ant = ant.upper()
+        at_date = cm_utils.get_astropytime(at_date).gps
+        aa_dict = {}
+        for val in self.session.query(cm_partconnect.AprioriAntenna).filter(func.upper(cm_partconnect.AprioriAntenna.antenna) == ant):
+            if val.start_gpstime < at_date:
+                aa_dict[val.start_gpstime] = val.status
+        try:
+            latest = sorted(aa_dict.keys())[-1]
+            return aa_dict[latest]
+        except IndexError:
+            return None
+
+    def get_apriori_status_set(self, at_date='now'):
+        ants = self._get_apriori_antenna_set()
+        aa_status = {}
+        for _x in cm_partconnect.get_apriori_antenna_status_enum():
+            aa_status[_x] = []
+
+        for a in ants:
+            msg = self.get_apriori_for(a, at_date=at_date)
+            if msg is not None:
+                aa_status[msg].append(a)
+        return aa_status
+
+    def get_dubitable_list(self, at_date='now'):
+        aa_status = self.get_apriori_status_set(at_date=at_date)
```

Comment:
I think it would be especially useful to have a docstring for this method, and with a description of the output format. Is it a comma-separated list like before? It would be good to know.